pwa-devdocs/src/technologies/overview/index.md

```diff
@@ -18,6 +18,13 @@ The following features define a basic PWA website:
 -   **Mobile "Install"** - Mobile users can add PWA sites to their home screens and even receive Push notifications from the site.
 -   **Shareable content** - Each page in a PWA site has a unique URL that can be shared with other apps or social media.
 
+This strategy is known as the RAIL model:
```

Comment:
would be useful to add a link to an external website for more information about this model